source/views/sis/course-search/detail/index.js

```diff
@@ -0,0 +1,146 @@
+// @flow
+
+import React from 'react'
+import type {CourseType} from '../../../../lib/course-search'
+import glamorous from 'glamorous-native'
+import {Badge} from '../../../building-hours/detail/badge'
+import {TableView, Section, Cell} from 'react-native-tableview-simple'
+import {convertTimeStringsToOfferings} from 'sto-sis-time-parser'
+import moment from 'moment-timezone'
+import {formatDay} from '../lib/format-day'
+import {
+	MultiLineDetailCell,
+	MultiLineLeftDetailCell,
+} from '../../components/multi-line-cell'
+import type {TopLevelViewPropsType} from '../../../types'
+import * as c from '../../../components/colors'
+
+const Container = glamorous.scrollView({
+	paddingVertical: 6,
+})
+
+const Header = glamorous.text({
+	fontSize: 36,
+	textAlign: 'center',
+	marginTop: 20,
+	marginHorizontal: 10,
+	color: c.black,
+})
+
+const SubHeader = glamorous.text({
+	fontSize: 21,
+	textAlign: 'center',
+	marginTop: 5,
+})
+
+const CellText = glamorous.text({
+	padding: 5,
+	color: c.black,
+})
+
+function Information({course}: {course: CourseType}) {
+	const profs = course.instructors ? course.instructors.join(', ') : null
+	const instructors = course.instructors ? (
+		<Cell cellStyle="LeftDetail" detail="Instructor(s)" title={profs} />
+	) : null
+	const gereqs = course.gereqs ? course.gereqs.join(', ') : null
+	const ges = course.gereqs ? (
+		<Cell cellStyle="LeftDetail" detail="GEs" title={gereqs} />
+	) : null
+	const credits = course.credits ? (
+		<Cell cellStyle="LeftDetail" detail="Credits" title={course.credits} />
+	) : null
+	const type = <Cell cellStyle="LeftDetail" detail="Type" title={course.type} />
+	const passFail = course.pn ? 'Yes' : 'No'
+	const pn = <Cell cellStyle="LeftDetail" detail="Pass/Fail" title={passFail} />
+	const prerequisites = course.prerequisites ? course.prerequisites : 'None'
+
+	const prereqs = (
+		<MultiLineLeftDetailCell detail="Prerequisites" title={prerequisites} />
+	)
+	return (
+		<Section header="COURSE INFORMATION" sectionTintColor={c.sectionBgColor}>
+			{instructors}
```

Comment:
I want to ask if you can try inlining all those earlier variables into this return statement…